### ISSUE TITLE
ScrollableViewport pauses animation when it changes size.

### ISSUE THREAD

**Issue by dragostis** [flutter/flutter#5478]
If a widget within a `ScrollableViewport` animates some layout change that results in the `ScrollableViewport` changing it size, any fling that happens to be running is paused and continued only after this animation is done.

@apwilson reported this bug earlier and @HansMuller added that the cause may likely be that the fling get continuously restarted while the animation is running.

**Comment by helinanever** [comment]
@dragostis can you confirm if this is still reproducible and if yes, could you provide a minimal complete reproducible code sample along with your `flutter doctor -v` and your
 `flutter run --verbose`

Thanks

**Comment by pedromassangocode** [comment]
Hi @dragostis
Even though the widget in question seems to not exist anymore, can you confirm if this is still reproducible and if yes, could you provide a minimal complete reproducible code sample along with your `flutter doctor -v`.
Thank you

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
